tokio-fs/src/file/create.rs

```diff
@@ -19,17 +22,18 @@ where
     }
 }
 
+impl<P> Unpin for CreateFuture<P> {} 
```

Comment:
do you want to just forward `Unpin` here aka `<P: Unpin>`?

Reply:
Kinda new to Pin/Unpin. Didn't really occur to me that this would satisfy the requirements for Unpin global implementations. :+1: